HISTORY.md

```diff
@@ -1,5 +1,7 @@
 ### mlpack ?.?.?
 ###### ????-??-??
+  * Added warm start feature to Random Forest (#2881).
```

Comment:
```suggestion
  * Added warm start feature to Random Forest (#2881); this feature is
    accessible from mlpack's bindings to different languages.
```
Just a suggestion---I wanted to make it clearer to users that they can use the warm start feature even if they are not writing C++. :)